p2p/src/node.rs

```diff
@@ -1416,7 +1416,9 @@ mod tests {
             let sql_pool = SqlitePool::connect_in_memory().await.unwrap();
 
             let metrics = Metrics::register(&mut prometheus_client::registry::Registry::default());
-            let store = Arc::new(EventService::try_new(sql_pool, true, true, vec![]).await?);
+            let store = Arc::new(
+                EventService::try_new(sql_pool, UndeliveredEventReview::Skip, true, vec![]).await?,
```

Comment:
I changed this behavior to skip ordering here but I don't think it is necessary for these tests.